specs/pvs/corla.pvs

```diff
@@ -470,6 +495,11 @@ ballot_manifests: THEORY
 BEGIN
   IMPORTING ballots, elections_equipment, information_systems
 
+% @todo kiniry Should this information also include what kind of
+% ballot each ballot is?  I.e., re-marked, phantom, etc.?
+% @todo kiniry These types will be refined when we review the
+% information provided by CDOS on 20 July 2017 about Dominion's file
+% formats.
   ballot_manifest_info: TYPE =
     [scanner_id_number, batch_number, batch_size, storage_container]
   ballot_manifest: TYPE = set[ballot_manifest_info]
```

Comment:
As I understand it, Dominion doesn't produce a ballot manifest. We've been very clear that the ballot manifest should come from a process completely separate from the scanners. The counties generate it according to our spec as a simple csv file. My csv sample above is based on the CDOS example from the DQ, about which they requested guidance in a place that I don't see offhand.